package.json

```diff
@@ -1,19 +1,24 @@
 {
-  "name": "express-api-starter",
+  "name": "message-system-api",
   "version": "1.0.0",
-  "description": "Starter project to get up and running with express quickly",
+  "description": "interview assignment from Ted & Gustaf",
+  "main": "server.js",
   "scripts": {
     "start": "babel-node server.js",
     "dev": "nodemon server.js --exec babel-node"
   },
-  "author": "",
+  "author": "Charlotte Holm",
```

Comment:
🤔 This should be your name, right?

Reply:
Yes, I used package.json from before when I was showing Charlotte how to build a project from scratch so I have all dependencies ready on the first installation and I forget to change Project ownership :D